Fix Aniwave : fix "Filemoon" support

MikeZeDev opened the pull request:
relevant commit https://github.com/manga-download/hakuneko-plugins/commit/1a8f9e425562d6634b8c0454f6d3f1328caa8321

A single line change tbh

MikeZeDev committed:
Fix Aniwave : fix "Filemoon" support

relevant commit

MikeZeDev committed:
FileMoon: add a delay to accomodate Aniwave proxy

W4stedWizard reviewed (approved):
Confirm fix runs perfectly on my Win10 machine (latest updates 23/03/2024)

MikeZeDev committed:
update from master (#6987)

* Fix ManhwaFreak: change fomain and filters (#6851)

Fixes https://github.com/manga-download/hakuneko/issues/6849

* Fix LyraScans: change domain & rebrand (#6857)

* Fix LyraScans: change domain & rebrand

Fixes https://github.com/manga-download/hakuneko/issues/6856

* update icon

* Fix NeoxScan: change domain (#6860)

* FIx ResetScans : change domain (#6854)

Fixes https://github.com/manga-download/hakuneko/issues/6853

* Fix SwatManga : change domain (#6844)

Fixes https://github.com/manga-download/hakuneko/issues/6842

* Fix KomikNesia : change domain (#6873)

Fixes https://github.com/manga-download/hakuneko/issues/6871

* Fix MangaToon: change CSS for chapters (#6887)

get rid of duplicate chapters

* Fix ComicK: update domain (#6897)

Fixes https://github.com/manga-download/hakuneko/issues/6896

* Fix MangaSail : change domain (#6898)

Fixes https://github.com/manga-download/hakuneko/issues/6894

* Fix MangaBat : change domain (#6899)

* Fix MangaStarz : change domain (#6901)

Fixes https://github.com/manga-download/hakuneko/issues/6900

* change domain (#6910)

Co-authored-by: Abdullah-KS <78895454+Abdullah-KS@users.noreply.github.com>

* FIx Siyahmelek: change domain (#6915)

Fixes https://github.com/manga-download/hakuneko/issues/6908

* Fix MangaTX : domain changed (#6916)

Fixes https://github.com/manga-download/hakuneko/issues/6911

* Fix TopToon : update CSS selectors (#6906)

* Fix TopToon : update CSS selectors

Fixes https://github.com/manga-download/hakuneko/issues/6905

* Update TopToon.mjs

* Update TopToon.mjs

* attempt to fix lint error

* dont use optional chaining

optional chaining doesnt work in stable build because electron version is too old

* Fix MagKan: use Comici Viewer (#6927)

Similar to YoungChampion, YoungAnimal, Comicride, ComicMedu, etc....

Fixes https://github.com/manga-download/hakuneko/issues/6926

* Fix winterscan : change domain (#6935)

https://github.com/manga-download/hakuneko/issues/6933

* fix MangaSect source (#6930)

* fix MangaSect source

* fix lint

* better mangasect fix

---------

Co-authored-by: Victor Abrate <me@vab.one>
Co-authored-by: MikeZeDev <MikeZeDev@users.noreply.github.com>

* Fix MoonWitchInLove : change domain (#6942)

Fixes https://github.com/manga-download/hakuneko/issues/6936

* add connector monzeekomik (#6954)

* add connector monzeekomik

* fix lint

---------

Co-authored-by: MikeZeDev <MikeZeDev@users.noreply.github.com>

* Fix ComicExtra : change domain (#6951)

* FIx KomikAV: domain type (#6881)

Fixes https://github.com/manga-download/hakuneko/issues/6879

* Fix Saikaiscan : change domain (#6963)

Fixes https://github.com/manga-download/hakuneko/issues/6620

* Fix LxHentai: change domain (#6970)

Fixes https://github.com/manga-download/hakuneko/issues/6968

* Lock ESLint Version (#6971)

* lock eslint version

* fix lint errors

---------

Co-authored-by: ZIRO <78895454+PD-ZIRO@users.noreply.github.com>
Co-authored-by: Abdullah-KS <78895454+Abdullah-KS@users.noreply.github.com>
Co-authored-by: Victor Abrate <victor.abrate@gmail.com>
Co-authored-by: Victor Abrate <me@vab.one>
Co-authored-by: akn19 <86169232+akn19@users.noreply.github.com>
Co-authored-by: Ronny <wegener.ronny@gmail.com>
